src/Abode.CLI/IDigitalTwinsClient.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Azure;
```

Comment:
Dont change anything with this PR but, something to think about for future. In an e2e twins solution, there would likely be multiple services needing to use the twins client (i.e. service for ingress (creating twins), one for reads/queries, one for managing models, potentially a CLI for internal ops like we have here. What might you do to facilitate reuse of this abstraction between all these services?